Makefile

```diff
@@ -3,8 +3,9 @@ JAIL_NIC?=vtnet0
 JAIL_IP?=172.16.0
 JAIL_NET?=16
 MYPYPATH = $(shell pwd)/.travis/mypy-stubs
-PYTHON ?= python3.6
 
+LATEST_PKG_PYTHON_VERSION != pkg info -g 'python3*' | cut -d'-' -f1 | sed 's/^python//' | sort -n | tail -n1 | sed -r 's/^([0-9])([0-9]+)/\1.\2/'
+PYTHON ?= python${LATEST_PKG_PYTHON_VERSION}
```

Comment:
this should still have a fallback, since we use this makefile on Travis, too.

Reply:
It is a bit complicated. There are multiple layers of overrides happening here.

```sh
PYTHON_VERSION ?= $(TRAVIS_PYTHON_VERSION)
SELECTED_PYTHON_VERSION != if [ "$(PYTHON_VERSION)" != "" ]; then echo $(PYTHON_VERSION); else pkg info -g 'python3*' | cut -d'-' -f1 | sed 's/^python//' | sort -n | tail -n1 | sed -r 's/^([0-9])([0-9]+)/\1.\2/'; fi
PYTHON ?= python${SELECTED_PYTHON_VERSION}
pyver= ${PYTHON:S/^python//:S/.//:C/\([0-9]+\)/\1/}
```

1.) When no `PYTHON_VERSION` was set, use `TRAVIS_PYTHON_VERSION`
2.) If neither was set, guess `SELECTED_PYTHON_VERSION` using pkg info
3.) This all can be overwritten via `PYTHON`.
4.) Read numeric `pyver` of whatever version was selected